Minio.core.nuspec

```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0"?>
-<package >
-  <metadata>
-    <id>Minio.NetCore</id>
-    <version>1.1.2</version>
-    <authors>Minio, Inc.</authors>
-    <owners>Minio, Inc</owners>
-    <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0.html</licenseUrl>
-    <projectUrl>https://github.com/minio/minio-dotnet</projectUrl>
-    <iconUrl>https://www.minio.io/logo/</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Minio .NET SDK for Amazon S3 Compatible Cloud Storage</description>
-    <releaseNotes>Rewrite of .NET sdk for support of .NetCore and .NET4.5.2 framework</releaseNotes>
-    <copyright>Copyright 2017</copyright>
-    <tags>minio cloud storage</tags>
-    <dependencies>
-      <group targetFramework=".NETStandard1.6">
-        <dependency id="RestSharp.NetCore" version="105.2.3" />
-        <dependency id="Newtonsoft.Json" version="10.0.2" />
-        <dependency id="System.Reactive.Linq" version="3.1.1" />
-        <dependency id="System.Runtime.Serialization.Xml" version="4.3.0" />
-        <dependency id="Microsoft.Win32.Registry" version="4.3.0" />
-        <dependency id="Microsoft.Build.Utilities.Core" version="15.1.548" />
-      </group>
-    </dependencies>
-    
-  </metadata>
-  <files>
-      <file src="Minio.Core\bin\Release.Core\**" target="lib" />
-  </files>
-</package>
```

Comment:
@niemyjski, this file is needed for creating nuget package - can you change the target framework, version (2.0.0) and update dependencies.